Production/python/Production.py

```diff
@@ -118,7 +123,8 @@
 objectdR = 0.5
 
 process.tauTupleProducer = cms.EDAnalyzer('TauTupleProducer',
-    isMC                            = cms.bool(not isData),
+    isMC                            = cms.bool(not isData if not isEmbedded else False),
+    isEmbedded                      = cms.bool(not isData if     isEmbedded else False),
```

Comment:
- Embedded should have `isMC` set to `True` in order to fill gen info => `isMC = cms.bool(not isData)`
- `isEmbedded = cms.bool(isEmbedded)`

Reply:
done baf22d5e70aa0cd424b3b7603e5a25036bb07361